src/LieTheory/WeylGroup.jl

```diff
@@ -753,6 +753,43 @@ function reflection(beta::RootSpaceElem)
   return W([list_of_indices; current_index; reverse(list_of_indices)])
 end
 
+###############################################################################
+# G-set functionality
+
```

Comment:
```suggestion

Base.:^(rw::Union{RootSpaceElem, WeightLatticeElem}, x::WeylGroupElem) = rw * x

```
playing a bit with all of this locally, this line seems to be needed to be able to e.g. call `orbits` for a block system. Could you also add a test for `orbits(bl)` where `bl` is the block system from the test file?

Reply:
I have now added a simple line which tests this by simply checking length of the orbits output.